runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -234,13 +251,12 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
      * @param usernameFile   username file path
      * @param passwordFile   password file path
      */
-    public NettyClientRouter(String host, Integer port, Boolean tls,
+    public NettyClientRouter(NodeLocator node, Boolean tls,
                              String keyStore, String ksPasswordFile, String trustStore,
                              String tsPasswordFile, Boolean saslPlainText, String usernameFile,
-                             String passwordFile, MetricRegistry metricRegistry) {
-        this.host = host;
-        this.port = port;
+                             String passwordFile) {
 
+        this.node = node;
         clientID = UUID.randomUUID();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "clientID"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)